sunbird-rc-esignet-integration-impl/src/main/java/io/mosip/esignet/sunbirdrc/integration/service/SunbirdRCVCIssuancePlugin.java

```diff
@@ -83,11 +83,11 @@ public class SunbirdRCVCIssuancePlugin implements VCIssuancePlugin {
     @Value("${mosip.esignet.vciplugin.sunbird-rc.issue-credential-url}")
     String issueCredentialUrl;
 
-    @Value("${mosip.esignet.authenticator.sunbird-rc.auth-factor.kba.registry-search-url}")
+    @Value("${mosip.esignet.vciplugin.sunbird-rc.credential-type.InsuranceCredential.registry-search-url}")
```

Comment:
this is specific to a credential type, we cannot use the @Value way, we have to take it through credentialTypeConfigMap

Reply:
removed